src/Components/forms/index.js

```diff
@@ -25,7 +25,7 @@ class Forms extends Component {
       const response = await axios.get(`${domain()}/teams`)
       const teamData = response.data.teams
       teamData.sort((a, b) => a.name.localeCompare(b.name))
-
+      teamData.push({ _id: '65035350fdbf584dc37a3383', name: 'I am not sure' })
```

Comment:
You don't need to add the `_id` since it's not and entry on the db , normally the ids are created automatically by mongodb.
```suggestion
      teamData.push({ name: 'I am not sure' })
```

Reply:
Done. Removed the id.